internal/handler/task.go

```diff
@@ -77,30 +95,76 @@ func (h *Handler) ModCmd() *cobra.Command {
 		Use:   "mod",
 		Short: "Modify a task by ID",
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Println("modifies task")
+			ti, err := parseTask(args, false)
+			if err != nil {
+				log.Fatal(err)
+			}
+			t, err := h.service.GetTask(*ti.id)
+			if err != nil {
+				return
+			}
+			//check if task exists, because of nil pointer dereference
+			if t == nil {
```

Comment:
We have to check, if task is not nil, because it will always throw "panic: runtime error: invalid memory address or nil pointer dereference"